pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -200,13 +200,17 @@ def test_setitem_extension_types(self, obj, dtype):
 
     @pytest.mark.parametrize(
         "ea_name",
-        # property would require instantiation
+        # mypy doesn't allow adding lists of different types
+        # https://github.com/python/mypy/issues/5492
         [
-            dtype.name
-            for dtype in ea_registry.dtypes
-            if not isinstance(dtype.name, property)
-        ]
-        + ["datetime64[ns, UTC]", "period[D]"],
+            *[
+                dtype.name
+                for dtype in ea_registry.dtypes
+                # property would require instantiation
+                if not isinstance(dtype.name, property)
+            ],
+            *["datetime64[ns, UTC]", "period[D]"],
```

Comment:
Sorry, but I find this ridiculous. And not you, but mypy ;) (since when are lists supposed to be of homogeneous content?) 
If mypy can't deal with it, I would just add a ignore comment, instead of the unpacking into a list

Reply:
+1

Reply:
Done.